### PR TITLE
feat(ledger-plugin): add signature/verification features into Validator/Verifier

### PR DESCRIPTION
**Pull request by takeutak**
This PR is related with Issue #441 .

Signed-off-by: Takuma TAKEUCHI <takeuchi.takuma@fujitsu.com>

### CODE REVIEW

**Comment by takeutak** [comment]
Contents:
- Add a new feature into Verifiers and Validators for signature and verification between their communication

The following README describes their boot method and how to use them
- https://github.com/takeutak/cactus/blob/feat_lp_add_signature_verification_features_into_Validator_Verifier/examples/cartrade/README.md

File: packages/routing-interface/util/RIFUtil.ts [hunk 1]
```diff
@@ -43,7 +43,7 @@ export class RIFUtil {
             return JSON.parse(jsonStr);
         }
         catch (error) {
-            logger.error(error)
+            // logger.error(error)
```
Comment (petermetz) [inline packages/routing-interface/util/RIFUtil.ts:46]: @takeutak 
I recommend disambiguation between legit JSON parsing errors and the case of the try block just running 

```typescript
JSON.parse("null")
```

which will return null the same way the catch block will. Currently there's no way for the caller to find out which one has happened.
With the logging commented out even if they are looking at the logs they won't be able to tell the difference.

File: packages/jest.config.js [hunk 2]
```diff
@@ -0,0 +1,201 @@
+/*
```
Comment (petermetz) [inline packages/jest.config.js:1]: @takeutak Is this file scoped for the entire project (e.g. all packages) or just a sub-set of packages?

cc: @sfuji822 @hartm @jonathan-m-hamilton

File: packages/routing-interface/util/RIFUtil.ts (hunk 1 above)
Comment (takeutak) [reply packages/routing-interface/util/RIFUtil.ts]: Thank you for your comment.  I corrected this file as your comment.
https://github.com/hyperledger/cactus/pull/442/commits/7c88afda62b140e9573f920a313e953215408058

File: packages/jest.config.js (hunk 2 above)
Comment (takeutak) [reply packages/jest.config.js]: Thank you for your suggestion.
As I said on the maintainer call, this file should be included by PR #444, not by this PR.  Then I deleted this file.
Moreover, I answered your question on the comment on PR #444.

**Comment by takeutak** [comment]
@petermetz I answered to your questions as the above. Would you mind if you confirm this pull-request again?

**Review by petermetz — APPROVED** [review]
LGTM

**Review by jonathan-m-hamilton — APPROVED** [review]
LGTM

**Pull request merged**